inject.js

```diff
@@ -292,7 +292,12 @@
         return;
       }
 
-      var regexp = new RegExp(escapeStringRegExp(match));
+      if (match.startsWith('/')) {
+        var regexp = new RegExp(match);
```

Comment:
If we don't validate and this throws an exception.. the whole init function will fail, right? I think we need to guard for this here, or validate at input time (harder, but better). 

Example here: https://stackoverflow.com/questions/17250815/how-to-check-if-the-input-string-is-a-valid-regular-expression

Reply:
You're probably right, I don't really know... I'll see if I can capture the error in the event the regex is invalid and direct it to pass to the next match.

Reply:
Should be fixed in commit https://github.com/igrigorik/videospeed/pull/546/commits/570c971f9a3a1d843c18b1e71b9cb410bc590f1b